internal/testrunner/runners/system/runner.go

```diff
@@ -981,7 +987,7 @@ func waitUntilTrue(fn func() (bool, error), timeout time.Duration) (bool, error)
 	timeoutTicker := time.NewTicker(timeout)
 	defer timeoutTicker.Stop()
 
-	retryTicker := time.NewTicker(1 * time.Second)
+	retryTicker := time.NewTicker(5 * time.Second)
```

Comment:
`waitUntilTrue` is used in more places and this change will affect them, could we maybe add the additional wait in the function to check the hits?

Reply:
Will send in a `retryTimeout` from the calling function

Reply:
Or Maybe a `time.sleep(4 * time.Second)` if config for `assert.hitCount` exists